src/components/scenes/NotificationCenterScene.tsx

```diff
@@ -61,11 +59,11 @@ export const NotificationCenterScene = (props: Props) => {
 
   const pinnedNotifInfoKeys = Object.keys(notifState)
     .filter(key => notifState[key].isPriority && !notifState[key].isCompleted)
-    .sort((a, b) => notifState[b].dateReceived - notifState[a].dateReceived)
+    .sort((a, b) => notifState[b].dateReceived.getMilliseconds() - notifState[a].dateReceived.getMilliseconds())
```

Comment:
Please use `Date.valueOf`, which returns the same sort of value as `Date.now`.

The `getMilliseconds` doesn't do what you think it does. In a date like `2025-01-01T02:03:04.345`, the `getMilliseconds` would return "345", which is just the decimal part of the seconds. So it's not very useful.